packages/web-components/docs/api-report.md

```diff
@@ -241,7 +241,7 @@ export const ambientShadow = "0 0 2px rgba(0, 0, 0, 0.14)";
 // @internal
 export class Anchor extends Anchor_2 {
     // @public
-    appearance: AnchorAppearance;
+    appearance?: AnchorAppearance;

```

Comment:
Curious as to what triggered this change? It seems like the overall changes in the PR shouldn't have affected the `web-components` package API in any way. This is also why the web-components folk's approval has been required for this PR to get merged in

Reply:
Thank you for getting back to me. Good question, I am unfamiliar with this package. I am going to cherry pick some of these changes into a fresh branch and see if this happens again.